src/main/java/org/hl7/davinci/priorauth/Endpoint.java

```diff
@@ -0,0 +1,122 @@
+package org.hl7.davinci.priorauth;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.UriInfo;
+
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Claim;
+import org.hl7.fhir.r4.model.ClaimResponse;
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.hl7.fhir.r4.model.OperationOutcome.IssueSeverity;
+import org.hl7.fhir.r4.model.OperationOutcome.IssueType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Endpoint {
+
+    static final Logger logger = LoggerFactory.getLogger(Endpoint.class);
+
+    static final Boolean requestXml = false;
+    static final Boolean requestJson = true;
+
+    static String REQUIRES_ID = "Instance ID is required: DELETE ClaimResponse?identifier=";
+    static String REQUIRES_PATIENT = "Patient Identifier is required: DELETE ClaimResponse?patient.identifier=";
+    static String DELETED_MSG = "Deleted ClaimResponse and all related and referenced resources.";
```

Comment:
Should this say ClaimResponse since delete could be called on any resource?

Reply:
Same with the two above it.

Reply:
Good catch. I copied these from the ClaimResponse endpoint so the names were hard coded. I generalized on my current branch